pydub/utils.py

```diff
@@ -276,11 +276,13 @@ def mediainfo_json(filepath, read_ahead_limit=-1):
     output = output.decode("utf-8", 'ignore')
     stderr = stderr.decode("utf-8", 'ignore')
 
-    info = json.loads(output)
-
-    if not info:
+    try:
+	info = json.loads(output)
```

Comment:
line 280 has a tab at the start instead of 4 spaces

(btw you saved my life thank you so much for this fix)

Reply:
Thanks!!